tests/integration/targets/inventory_terraform_state_google/aliases

```diff
@@ -0,0 +1 @@
+cloud/gcp
```

Comment:
Should we add `unsupported` here?

Reply:
it is automatically unsupported, I would rather keep this like this so it can be run locally when the user defines the `tests/integration/cloud-config-gcp.ini` file